official/utils/logging/logger_test.py

```diff
@@ -25,16 +25,22 @@
 import unittest
 
 import tensorflow as tf  # pylint: disable=g-bad-import-order
-from tensorflow.python.client import device_lib
 
 from official.utils.logging import logger
 
 
 class BenchmarkLoggerTest(tf.test.TestCase):
 
+  def setUp(self):
+    super(BenchmarkLoggerTest, self).setUp()
+    self.original_environ = dict(os.environ)
```

Comment:
Could you put a comment about why this was necessary and when others would need to worry about environment variables?

Reply:
Done.